cmd/oc/oc.go

```diff
@@ -20,6 +21,10 @@ import (
 )
 
 func main() {
+	if reexec.Init() {
```

Comment:
In `oc`?  This doesn't look right.  If you must have it for a specific command, localize the change there.